src/socket/socket.js

```diff
@@ -86,19 +84,59 @@ export function initializeSocket(httpServer) {
         return;
       }
       // Example usage of modifyPoints function
-     
+
       const leaderboard = generateLeaderboard(roomId);
       io.to(socket.id).emit("leaderboard", leaderboard);
     });
 
+    let imageUploadOrder = [];
```

Comment:
what i'm thinking you're trying to implement here is an array that stores images from all the players when the match/round is over. 

here's why it wouldn't work: this is all being done in the io object, and it runs this callback function (line 21) for every user that makes a connection, so it will create a different array for each user. (ask me if you need a little more explanation or if i got something wrong).

Reply:
i will try to update it